wp-web-push/wp-web-push-main.php

```diff
@@ -89,6 +97,51 @@ public static function on_transition_post_status($new_status, $old_status, $post
       }
     }
   }
+
+  public static function get_trigger_by_key_value($key, $value) {
+    foreach(self::$ALLOWED_TRIGGERS as $trigger) {
+      if($trigger[$key] === $value) {
+        return $trigger;
+      }
+    }
+    return false;
+  }
+
+  public static function get_triggers_by_key_value($key, $value) {
+    $matches = array();
+
+    foreach(self::$ALLOWED_TRIGGERS as $trigger) {
+      if($trigger[$key] === $value) {
+        $matches[] = $trigger;
+      }
+    }
+    return $matches;
+  }
+
+  private static function add_trigger_handlers() {
+    $enabled_triggers = get_option('webpush_triggers');
+    foreach($enabled_triggers as $trigger) {
+        //echo 'attempting to find trigger for: '.$trigger.'<br>';
+        $trigger_detail = self::get_trigger_by_key_value('key', $trigger);
+        if($trigger_detail) {
+          add_action($trigger_detail['hook'], array($this, $trigger_detail['action']), 10, 2);
```

Comment:
My debugging is confirming the proper actions are being added but for some reason the functions below don't trigger.  :(